CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Added:
 * Add v1 `chronosphere_logging_allocation_config` resource
 * Add support for `is_root_span` field in common trace span filter type.
 * Allow disabling dry-run via provider configuration `disable_dryrun`.
+* Adds `priority_thresholds` to `chronosphere_resource_pools_config` pools.
```

Comment:
this should move into the Unreleased section, not in the 1.7.0 sections since that is already released